README.md

```diff
@@ -89,7 +89,7 @@ create_vector_index(
     driver,
     INDEX_NAME,
     label="Document",
-    property="textProperty",
+    embedding_property="textProperty",
```

Comment:
Might be worth renaming "textProperty" to something like "vectorProperty" to make it clear that this is the property on the node that'll hold the vector and not the text property that'll be used to create the vectors